android/PhysicalWeb/app/src/main/res/values/strings.xml

```diff
@@ -62,11 +54,6 @@
     <string name="title_activity_oob">OobActivity</string>
```

Comment:
Should this be here?  If it's supposed to be for humans to read, maybe we should change the string.